### DISCUSSION TITLE
Difference between linux_amd64.tar.gz and linux_amd64_full.tar.gz?

### DISCUSSION THREAD

**Discussion by ericlingit** [seaweedfs/seaweedfs#4197] [Q&A] [answered]
What's the difference between the normal tar.gz file and the other with 'full' in its name?

**Comment by ericlingit** [comment, marked as answer]
tl;dr: the `full` version includes support for elastic, ydb, gocdk, and tikv (presumably for the filer).

Since this difference isn't mentioned anywhere in the Wiki or README, I dug through many pages of release notes before noticing the difference in their GitHub action build scripts:
- release note for [3.00](https://github.com/seaweedfs/seaweedfs/releases/tag/3.00) mentioned
    > trim out uncommonly used libraries with large size when compiled, including: "elastic, gocdk, sqlite, hdfs".
    - but does not mention a 'full' binary build
    - a `linux_amd64_full.tar.gz` doesn't show up in the asset list until release 3.06
- The clue is in the build scripts:
    - [.github/workflows/binaries_release1.yml](https://github.com/seaweedfs/seaweedfs/blob/master/.github/workflows/binaries_release1.yml) and [.github/workflows/binaries_release4.yml](https://github.com/seaweedfs/seaweedfs/blob/master/.github/workflows/binaries_release4.yml)
    - one has `build_flags: -tags elastic,ydb,gocdk,tikv`, while the other doesn't
